.github/workflows/pre-commit.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Install dependencies
         run: |
               python -m pip install --upgrade pip
-              pip install -e .[dev]
+              pip install 'pre-commit>=2.10.1' 'flake8'
```

Comment:
I guess you'd just need pre-commit... IIUC the lint check is initiated in `run_tests.yml`